src/components/motor_output_reordering/MotorOutputReorderingCanvas.js

```diff
@@ -0,0 +1,271 @@
+'use strict';
+
+class MotorOutputReorderCanvas
+{
+    constructor(canvas, droneConfiguration, motorClickCallback, spinMotorCallback)
+    {
```

Comment:
I prefer the `{` in the same line than the declaration of the method or the class, but I think we have no rule about this... @mikeller do we have?
I've seen you use both ways in the code, maybe by mistake.

Reply:
@McGiverGim, thanks for looking into my code i really appreciate it!

Somewhere the betaflight/betaflight coding style i've seen that for functions it should be 
```
function abs()
{
}
```
but for the if/while/for it should be
```
while {
}
```
But maybe that's just for C code?
https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md

Reply:
Then I'm wrong. We try to maintain the most similar both parts, to help developers to code for both parts in the same way, so your code is ok. I will try to change it in my code for future PR 😄 

The file `MotorOutputReorderingComponent.js` has a mixed way of functions, take a look to them to let all of them in the correct way.